### ISSUE TITLE
ability to use on mobile

### ISSUE THREAD

**Issue by saad863** [XengShi/materialYouNewTab#17]
is there any this can be used on mobile

**Comment by XengShi** [comment]
Wait for next update!

**Comment by prem12321kumar** [comment]
@itz-rj-here 
There is an option of adding extension (Add-ons) too in Firefox and Edge for Android version.
But not in Chrome.

@saad863 
One way I can think of is
Goto https://xengshi.github.io/materialYouNewTab/
From 3 dots (Menu), tap 'Add to Home Screen'
Then use this shortcut instead of chrome or others. 
Kinda works. Temporary solution.

**Comment by XengShi** [comment]
> @itz-rj-here 
> There is an option of adding extension (Add-ons) too in Firefox and Edge for Android version.
> But not in Chrome.
> 
> @saad863 
> One way I can think of is
> Goto https://xengshi.github.io/materialYouNewTab/
> From 3 dots (Menu), tap 'Add to Home Screen'
> Then use this shortcut instead of chrome or others. 
> Kinda works. Temporary solution. 

Yes, but currently our extension is not compatible / responsive for mobile devices. Some modifications are needed in style.css

**Comment by XengShi** [comment]
> > @itz-rj-here 
> > There is an option of adding extension (Add-ons) too in Firefox and Edge for Android version.
> > But not in Chrome.
> > 
> > @saad863 
> > One way I can think of is
> > Goto https://xengshi.github.io/materialYouNewTab/
> > From 3 dots (Menu), tap 'Add to Home Screen'
> > Then use this shortcut instead of chrome or others. 
> > Kinda works. Temporary solution. 
> 
> Yes, but currently our extension is not compatible / responsive for mobile devices. Some modifications are needed in style.css


--------

For setting it on mobile phone.

(CHROME) -->
Click (•••) dots
Click ⚙️ Settings 
Then 'Homepage' 
Enable the toggle and set

This url
https://xengshi.github.io/materialYouNewTab/

For Homepage

**Comment by prem12321kumar** [comment]
> > Yes, but currently our extension is not compatible / responsive for mobile devices. Some modifications are needed in style.css
> 

Yep, true

> For setting it on mobile phone.
> 
> (CHROME) --> Click (•••) dots Click ⚙️ Settings Then 'Homepage' Enable the toggle and set
> 
> This url https://xengshi.github.io/materialYouNewTab/
> 
> For Homepage

Ah yes, I forgot there's this option in android too.

**Comment by itz-rj-here** [comment]
Ohhh I see
